universal-login-contracts/test/contracts/ERC1077.js

```diff
@@ -16,6 +17,18 @@ const {gasPrice} = DEFAULT_PAYMENT_OPTIONS;
 const overrideOptions = {gasPrice, gasLimit: 200000};
 
 describe('ERC1077', async  () => {
+  describe('Void ERC1077', () => {
```

Comment:
We should:
- push that describe to the top level, 
- make sure the signature is empty
- create message like this: ` msg = {...transferMessage, from: identityWithZeroKey.address};`